### ISSUE TITLE
[Bug]: Numbered/ordered lists are renedered as bulleted/unordered lists

### ISSUE THREAD

**Issue by daenney** [GeopJr/Tuba#480]
### Describe the bug

When a toot includes an ordered list, Tuba renders it as an unordered list.

For example: https://ap.dny.social/@ohno/statuses/01H9BFC94CTVMWA96D5925GXP2.

When viewed on the web and in other clients (for example FediText), I correctly get an ordered list with 1., 2. etc. However, when viewing that same status in Tuba, it's rendered as an unordered/bulleted list instead.

### Steps To Reproduce

1. View a post (written in Markdown) with an ordered list

### Logs and/or Screenshots

![Screenshot from 2023-09-03 17-09-59](https://github.com/GeopJr/Tuba/assets/569574/b52e8348-6ea1-4ca6-8791-41ae3a407430)

### Instance Backend

GoToSocial

### Operating System

Arch Linux

### Package

Flatpak

### Troubleshooting information

```
os: GNOME 44 (Flatpak runtime)
prefix: /app
flatpak: true
version: 0.4.1 (production)
gtk: 4.10.4 (4.10.4)
libadwaita: 1.3.3 (1.3.3)
libsoup: 3.4.2 (3.4.2)
libgtksourceview: 5.8.0 (5.8.0)
```

**Comment by GeopJr** [comment]
Nice catch! It should be fixed as of #482!

**Comment by daenney** [comment]
Damn that was quick :rocket:. Thanks so much!
